bc_obps/registration/schema/v1/contact.py

```diff
@@ -1,18 +1,44 @@
 from typing import Optional
+from registration.models.event.transfer_event import TransferEvent
+from registration.models.operation import Operation
+from registration.models.operator import Operator
 from ninja import ModelSchema, Field, FilterSchema
 from registration.models import Contact
 
+from django.db.models import ManyToManyRel, ManyToOneRel
+
 
 class ContactOut(ModelSchema):
     street_address: Optional[str] = Field(None, alias="address.street_address")
     municipality: Optional[str] = Field(None, alias="address.municipality")
     province: Optional[str] = Field(None, alias="address.province")
     postal_code: Optional[str] = Field(None, alias="address.postal_code")
+    places_assigned: Optional[list] = []
 
     @staticmethod
     def resolve_phone_number(obj: Contact) -> str:
         return str(obj.phone_number)
 
+    @staticmethod
+    def resolve_places_assigned(contact: Contact) -> list:
+        related_fields = []
+        for contact_field in contact._meta.get_fields():
+            if isinstance(contact_field, (ManyToOneRel, ManyToManyRel)):
```

Comment:
There's also the ManyToMany**Field**, and that gives us stuff like contact documents. I don't think users can change that, so I don't know that it's useful to show here, but they won't be able to delete if PROTECT is on and a contact is being used somewhere. Thoughts?